es/frontend.php

```diff
@@ -98,4 +103,49 @@
     'settingsChanged' => 'Configuraciones cambiadas durante la partida, la repetición será invalidada.',
     'segment' => 'Segmento',
     'duration' => 'Duración',
+    'roomFull' => 'Parece que la sala esta llena',
+    'lobbyInfo' => 'Este chat es compartido con {discord}.',
+    'newHost' => 'Ahora sos el anfitrión de esta sala.',
+    'badRoom' => 'Esta sala no existe más, ingresando a la sala Default!',
+    'stngsChanged' => 'Configuraciones cambiadas por el anfitrión',
+    'stngsCustom' => 'Configuraciones personalizadas',
+    'aSpec' => 'Ya estás en modo espectador!',
+    'aPlay' => 'Ya estás jugando!',
+    'repFail' => 'La repetición no se pudo cambiar.',
+    'repInChat' => 'Se dumpeo la repetición en el chat. Para conservarlo copia el texto del cuadro y pegalo en tu editor de texto favorito.',
+    'repTxtInfo' => 'La copia del archivo de repetición puede ser reproducida en el reproductor interno o posiblemente resubida al sitio.',
+	
+    'newPB' => 'TU NUEVA MARCA PERSONAL',
+    'firstPB' => 'Este fue tu primer juego. Haz otra marca personal para hacer un seguimiento de las estadisticas de mejora.',
+    'infoPB' => 'Tu anterior record era {prevPB} logrado {prevAgo}. La mejora es {PBdiff}.',
+    'daysAgo' => 'días atras',
+
+    'raceFin' => 'LA CARRERA TERMINO',
+    'raceFinInfo' => 'Puedes completar la carrera, pero la proxima ronda puede empezar en cualquier momento.',
+
+    'notTSD' => 'NO TSD',
+    'notTSDInfo' => 'Solamente es permitido el T-Spin Double',
+	
+    'oops' => 'Uf!',
+    'chatNA' => 'El chat publico no esta habilitado para invitados o usuarios con menos de {chReq} horas de juego.',
+    'leMore' => 'Aprende más',
+
+    'connLimit' => 'Las conexiones máximas para esta IP ha sido alcanzada. Si necesitas incrementar este limite contactanos por Discord',
+    'idleDC' => 'Desconectado por inactividad! La zona de espectadores esta llena.',
+    'RLreach' => 'Limite de velocidad alcanzado.',
+    'ban1' => 'El acceso a los juegos ha sido restringido permanentemente. Todavía puedes jugar los modos de un solo jugador.',
+    'ban2' => 'This user account is BANNED. Relogin to the website for more information.',
```

Comment:
English version can be removed, or provide translation

Reply:
My fault, I did not see it, **fixed**